guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,8 +4,26 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+=== Plugin translation infrastructure in frontend
+
+With this release it's possible for plugins to use translations in Javascript.
+Since the front end is largely moving towards Javascript, this is a great improvement for who care about an internationalized application.
+Note that having the infrastructure in place doesn't guarantee everything is translated.
+
+https://projects.theforeman.org/issues/36330[#36330]
```

Comment:
Not sure I'm too happy about this, but I'm goign with this so I can announce the release candidate.